src/ui/views/User/User.jsx

```diff
@@ -35,9 +35,11 @@ export default function Dashboard(props) {
   const navigate = useNavigate();
 
   useEffect(() => {
-    const id = props.match.params.id;
-    if (id == null) {
+    let id = null;
+    if (!props.match) {
       setIsProfile(true);
+    } else {
+      id = props.match.params.id;
```

Comment:
`props` doesn't exist here anymore. The `useParams()` function is required from `react-router-dom` 👍

Reply:
Is this what you mean?

```
  const params = useParams();

  useEffect(() => {
    let id = null;
    if (!params.match) {
      setIsProfile(true);
    } else {
      id = params.match.params.id;
    }
```

Reply:
saw your fix on the reconcile branch. Thank you!